identity/generate.go

```diff
@@ -58,7 +58,6 @@ func GetPublicIdentity(b64Identity string) (*string, error) {
 
 	if publicIdentity.Target != "user" {
 		if publicIdentity.Target == "email" {
-			publicIdentity.Target = "hashed_email"
```

Comment:
why?

Reply:
Since we switched to the systematic naming:
email -> hashed_email  
phone_number -> hashed_phone_number

Now we can just do: `publicIdentity.Target = "hashed_" + publicIdentity.Target`